src/Cake.Core/Constants.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Cake.Core
+{
+    internal static class Constants
+    {
+        public static class Paths
```

Comment:
@patriksvensson I general, this is a good idea.  Should we create an issue for adding in the existing Configuration values as well?  There is only 1 I think, but think they should be in the same place.